prost-derive/src/lib.rs

```diff
@@ -304,6 +305,7 @@ fn try_enumeration(input: TokenStream) -> Result<TokenStream, Error> {
             }
         }
 
+        #[allow(deprecated)]
```

Comment:
Will this hide deprecation warnings in non deprecated fields?

Reply:
fix.